src/transformers/models/data2vec/modeling_data2vec.py

```diff
@@ -45,6 +45,13 @@
     find_pruneable_heads_and_indices,
     prune_linear_layer,
 )
+from ...models.wav2vec2.modeling_wav2vec2 import (
```

Comment:
Do we want to copy these over, or importing them from `modeling_wav2vec2` is fine? @patrickvonplaten

Reply:
We should copy all of them over with Copied from statements ;-)